internal/runners/commit/commit.go

```diff
@@ -159,7 +160,19 @@ func (c *Commit) Run() (rerr error) {
 	dependencies.OutputChangeSummary(out, rtCommit.BuildPlan(), oldCommit.BuildPlan())
 
 	// Report CVEs.
-	if err := cves.NewCveReport(c.prime).Report(rtCommit.BuildPlan(), oldCommit.BuildPlan()); err != nil {
+	newReqs, err := newRequirements(oldCommit.BuildScript(), rtCommit.BuildScript())
```

Comment:
This doesn't feel right, because we're going to need to duplicate this logic in other places that will eventually get CVE reporting. It looks like `state import` is not passing any names to `cves.NewCveReport()` either, so presumably it suffers from the same bug.

Can we put all of this logic in the `Report()` method? Kind of like how the change summary reporter diffs two buildplans? (https://github.com/ActiveState/cli/blob/0bb4d847b896c83a4d31cbba042b8ba65b06c933/internal/runbits/dependencies/changesummary.go#L30). If we need absolutely need buildscripts, then maybe we should be passing commits to the method instead of buildplans.